board-games/test/player-test.js

```diff
@@ -0,0 +1,128 @@
+
+    assert.equal(player.collection.length, 2);
+    assert.equal(player.collection[1].name, 'Dominion');
+  });
+
+  it.skip('should be able to add games to the collection', function() {
+    var game1 = new Game({name: 'Gloomhaven'});
+    var game2 = new Game({name: 'Spirit Island'});
+    var game3 = new Game({name: 'Scythe'});
+    var player = new Player({name: 'Zane', collection: []});
+
+    assert.deepEqual(player.collection, []);
+
+    player.addToCollection(game1);
+
+    assert.equal(player.collection.length, 1);
+    assert.equal(player.collection[0].name, 'Gloomhaven')
+
+    player.addToCollection(game2);
+    player.addToCollection(game3);
+
+    assert.equal(player.collection.length, 3);
+    assert.equal(player.collection[1].name, 'Spirit Island');
+    assert.equal(player.collection[2].name, 'Scythe');
+  });
+
+  it.skip('should be able to have a top 3 games', function() {
+    var game1 = new Game({name: 'Gaia Project'});
+    var game2 = new Game({name: 'Dune Imperium'});
+    var game3 = new Game({name: 'Terraforming Mars'});
+    var player = new Player({name: 'Phil'});
+
+    player.addToCollection(game1);
+    player.addToCollection(game2);
+    player.addToCollection(game3);
+    player.declareFavorites('Gaia Project', 'Dune Imperium', 'Terraforming Mars');
+
+    assert.equal(player.favoriteGames.firstFavorite, 'Gaia Project');
+    assert.equal(player.favoriteGames.secondFavorite, 'Dune Imperium');
+    assert.equal(player.favoriteGames.thirdFavorite, 'Terraforming Mars');
+  });
+
+  it.skip('should not be able to declare a favorite unless it is in thier collection', function(){
+    var game = new Game({name: 'Brass Birmingham'});
+    var player = new Player({name: 'Sophia'});
+
+    player.declareFavorites('Brass Birmingham');
+
+    assert.deepEqual(player.favoriteGames, {});
+
+    player.addToCollection(game);
+    player.declareFavorites('Brass Birmingham');
+    assert.equal(player.favoriteGames.firstFavorite, 'Brass Birmingham');
+  });
+
+  it.skip('should be able to take games to game night', function() {
+    var player = new Player({name: 'Megan'});
+    var game1 = new Game({name: 'Codenames'});
+    var game2 = new Game({name: 'Ticket to Ride'});
+    player.addToCollection(game1);
+    player.addToCollection(game2);
+    player.takeToGameNight([game1, game2]);
+
+    assert.equal(player.gameNightGames.length, 2);
+    assert.equal(player.gameNightGames[0].name, 'Codenames');
+  });
+
+  it.skip('should only be able to take games that are in thier collection', function() {
+    var player = new Player({name: 'Michael'});
+    var game1 = new Game({name: 'Sagrada'});
+    var game2 = new Game({name: 'Splendor'});
+    player.addToCollection(game2);
+    player.takeToGameNight([game1, game2]);
+
+    assert.equal(player.gameNightGames.length, 1);
+    assert.equal(player.gameNightGames[0].name, 'Splendor');
+  });
+
+  it.skip('should be able to have an experience level', function(){
```

Comment:
Maybe re-word to "Should start out with no experience?"

Reply:
Done